### PR TITLE
feat: add video microdata

### PR DESCRIPTION
**Pull request by aeksandla**

### CODE REVIEW

File: src/sub-blocks/LayoutItem/LayoutItem.tsx [hunk 1]
```diff
@@ -43,6 +44,13 @@ const LayoutItem = ({
             return null;
         }
         const themedMedia = getThemedValue(media, theme);
+        const mediaWithMicrodata = mergeVideoMicrodata(themedMedia, {
+            name:
+                typeof content.title === 'string'
```
Comment (NikitaCG) [inline src/sub-blocks/LayoutItem/LayoutItem.tsx:49]: Are we can create deserialization on this step for `content`?

File: src/components/Media/Media.tsx [hunk 2]
```diff
@@ -155,8 +158,24 @@ export const Media = (props: MediaAllProps) => {
...
     ]);
 
+    const videoMicrodataScript = useMemo(() => {
+        const json = JSON.stringify({
+            '@context': 'http://schema.org/',
+            '@type': 'VideoObject',
+            uploadDate: microdata?.contentUpdatedDate,
+            contentUrl: video?.src?.[0] || videoIframe || youtube,
+            thumbnailUrl: previewImg,
+            ...videoMicrodata,
```
Comment (NikitaCG) [inline src/components/Media/Media.tsx:168]: maybe we need to use `sanitizeMicrodata` only in this place ?

**Review by NikitaCG — CHANGES_REQUESTED** [review]

File: src/components/Media/Media.tsx (hunk 2 above)
Comment (aeksandla) [reply src/components/Media/Media.tsx]: fixed

File: src/sub-blocks/LayoutItem/LayoutItem.tsx (hunk 1 above)
Comment (aeksandla) [reply src/sub-blocks/LayoutItem/LayoutItem.tsx]: fixed

**Review by NikitaCG — APPROVED** [review]

**Pull request merged**